<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Buttons</class>
 <widget class="QWidget" name="Buttons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>681</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>101</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QCheckBox" name="checkBoxA">
      <property name="text">
       <string>CheckBoxA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxB">
      <property name="text">
       <string>CheckBoxB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBoxC">
      <property name="text">
       <string>CheckBoxC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>125</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radioButtonA">
      <property name="text">
       <string>RadioButtonA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonB">
      <property name="text">
       <string>RadioButtonB</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonC">
      <property name="text">
       <string>RadioButtonC</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>110</y>
     <width>125</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButton1">
      <property name="text">
       <string>RadioButton1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton2">
      <property name="text">
       <string>RadioButton2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton3">
      <property name="text">
       <string>RadioButton3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>110</y>
     <width>101</width>
     <height>73</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QCheckBox" name="checkBox1">
      <property name="text">
       <string>CheckBox1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox2">
      <property name="text">
       <string>CheckBox2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox3">
      <property name="text">
       <string>CheckBox3</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
